src/bp/ui-studio/src/web/views/OneFlow/diagram/nodes/Components/NodeHeader.tsx

```diff
@@ -60,9 +60,9 @@ const NodeHeader: FC<Props> = ({
       {!isEditing ? (
         <Button
           icon={setExpanded ? icon : null}
-          onClick={() => setExpanded?.(!expanded)}
+          onClick={() => setExpanded && setExpanded(!expanded)}
           className={style.button}
-          onContextMenu={e => handleContextMenu?.(e)}
+          onContextMenu={e => handleContextMenu && handleContextMenu(e)}
```

Comment:
Just wondering why the change, it wasn't working ?

Reply:
Since all the nodes were handled in the same file I had to add this condition when passing the context menu ```hasContextMenu && handleContextMenu``` which made it to not be undefined, but falsy instead so whenever the `handleContextMenu` was called when it shouldn't be available it was trying to call a function on `false` and created an error